app/serializers/rest/reaction_serializer.rb

```diff
@@ -21,11 +21,25 @@ def custom_emoji?
     object.custom_emoji.present?
   end
 
+  def name
```

Comment:
note: since this is editing the reaction serializer (that is also used for announcements), could you verify that it doesnt break or bug the announcement reaction list?

Reply:
They seem good here; this announcement was made with the patch applied.
![image](https://github.com/glitch-soc/mastodon/assets/21112968/8dfc7ccc-1943-4428-af6d-4b5bbefafaf6)